src/component/useFileUpload.js

```diff
@@ -60,14 +64,25 @@ const simpleAction = (type, dispatch) => dispatch({type})
 export function useFileUpload({
   reducer = dragDropReducer,
   customValidation = () => {},
-  validate,
   ...props
 } = {}) {
   const {current: initialState} = useRef({...defaultState, ...props})
   const fileRef = useRef(null)
   const [state, dispatch] = useReducer(reducer, initialState)
-  const {files, multiple, filecountlimit, acceptableextensions} = state
+  const {
+    files,
+    multiple,
+    filecountlimit,
+    acceptableextensions,
+    previewCustomStyle,
+  } = state
   const register = ref => (fileRef.current = ref)
+  const {previewFiles} = usePreviewFiles(files, previewCustomStyle)
```

Comment:
I do not see any usecases of this.. i think this one we can simply export it. As we are exporting files array for the user component this can be used as a utitlity in the examples.

Reply:
You mean this also can be utilized as separate hook to configure ??

Reply:
No.  we dont need this in the lib. we simply export and import in examples.